### PR TITLE
Enqueue Customizer styles via the new filter in Common.

### PR DESCRIPTION
**Pull request by Camwyn**
⚠️ Requires: https://github.com/moderntribe/tribe-common/pull/1352

[TEC-3401]

[TEC-3401]: https://moderntribe.atlassian.net/browse/TEC-3401

### CODE REVIEW

File: src/Tribe/Assets.php [hunk 1]
```diff
@@ -774,4 +775,19 @@ public function get_js_dynamic_data() {
 		return $data;
 	}
 
+	/**
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:778]: :no_entry_sign: Doc comment for parameter "$css_template" missing

<sub>hash: a51411238bd952ceabe75f32aec2b82e</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 2]
```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:719]: :no_entry_sign: Doc comment for parameter "$css_template" missing

<sub>hash: 63da2489f20fe951a0b5f8ae4a07d85d</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 3]
```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
...
+	 * @return array Modified array of sheets to search for.
+	 */
+	public function customizer_inline_stylesheets( $sheets, $css_template ) {
+		$v2_sheets = [
+			'tribe-events-views-v2-skeleton',
+			'tribe-events-views-v2-full',
+		];
+
+		// Unenqueue legacy sheets.
+		if ( ( $keys = array_keys( $sheets, 'tribe-events-calendar-style' ) ) !== [] ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:734]: :no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 9f8c7db739dcf9dc1e278116c833bdbd</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 4]
```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
...
+			foreach( $keys as $key ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:735]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after FOREACH keyword; 0 found
* :no_entry_sign: There must be a space between foreach and the first parenthesis '('

<sub>hash: fe7a821778db6a04a5a8e8afaf9140e4</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 5]
```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
...
+				unset( $sheets[$key] );
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:736]: * :no_entry_sign: Array keys must be surrounded by spaces unless they contain a string or an integer.
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: 307b1461285168303449b10a2c09702a</sub>

File: src/Tribe/Assets.php [hunk 6]
```diff
@@ -774,4 +775,20 @@ public function get_js_dynamic_data() {
...
+	 * Add legacy stylesheets to customizer styles array to check.
+	 *
+	 * @param array $sheets Array of sheets to search for.
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:781]: :no_entry_sign: Expected 2 spaces after parameter type; 1 found

<sub>hash: c7f61054e6b9901ca170e15a8e2e03f9</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 7]
```diff
@@ -713,4 +715,30 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @param array $sheets Array of sheets to search for.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:723]: :no_entry_sign: Expected 2 spaces after parameter type; 1 found

<sub>hash: 0e0bc55c663ae7bd452ab9d443917264</sub>

File: src/Tribe/Assets.php [hunk 8]
```diff
@@ -774,4 +775,20 @@ public function get_js_dynamic_data() {
 		return $data;
 	}
 
+	/**
+	 * Add legacy stylesheets to customizer styles array to check.
+	 *
+	 * @param array  $sheets Array of sheets to search for.
+	 * @param string  $css_template String containing the inline css to add.
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:782]: :no_entry_sign: Expected 1 spaces after parameter type; 2 found

<sub>hash: c6aa06f98c366f40440876997fe50cf5</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 9]
```diff
@@ -713,4 +715,30 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @param array  $sheets Array of sheets to search for.
+	 * @param string  $css_template String containing the inline css to add.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:724]: :no_entry_sign: Expected 1 spaces after parameter type; 2 found

<sub>hash: 5ee35cb16e6aef6457e02876abef50ef</sub>

File: src/Tribe/Assets.php [hunk 10]
```diff
@@ -774,4 +775,20 @@ public function get_js_dynamic_data() {
 		return $data;
 	}
 
+	/**
+	 * Add legacy stylesheets to customizer styles array to check.
+	 *
```
Comment (bordoni) [inline src/Tribe/Assets.php:780]:
```suggestion
	 *
	 * @since TBD
	 *
```

File: src/Tribe/Views/V2/Hooks.php [hunk 11]
```diff
@@ -713,4 +715,30 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
```
Comment (bordoni) [inline src/Tribe/Views/V2/Hooks.php:722]:
```suggestion
	 *
	 * @since TBD
	 *
```

File: src/Tribe/Views/V2/Hooks.php [hunk 12]
```diff
@@ -713,4 +715,32 @@ public function print_single_json_ld() {
...
+	 * @since TBD
+	 *
+	 * @param array  $sheets Array of sheets to search for.
```
Comment (lucatume) [inline src/Tribe/Views/V2/Hooks.php:725]:
```suggestion
	 * @param array<strings>  $sheets Array of sheets to search for.
```

File: src/Tribe/Views/V2/Hooks.php [hunk 13]
```diff
@@ -713,4 +715,32 @@ public function print_single_json_ld() {
...
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string>  $sheets Array of sheets to search for.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:725]: :no_entry_sign: Expected 1 spaces after parameter type; 2 found

<sub>hash: 7999932993d21446b2c4420dcc4bfbb7</sub>

File: src/Tribe/Views/V2/Hooks.php [hunk 14]
```diff
@@ -713,4 +715,32 @@ public function print_single_json_ld() {
...
+	 * @param string $css_template String containing the inline css to add.
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Hooks.php:726]: :no_entry_sign: Expected 8 spaces after parameter type; 1 found

<sub>hash: 8b5ef49ac061f39e697a9182e15fc049</sub>

File: src/Tribe/Views/V2/Hooks.php (hunk 13 above)
Comment (Camwyn) [inline src/Tribe/Views/V2/Hooks.php:725]:
```suggestion
	 * @param array<string> $sheets Array of sheets to search for.
```

File: src/Tribe/Views/V2/Hooks.php (hunk 14 above)
Comment (Camwyn) [inline src/Tribe/Views/V2/Hooks.php:726]:
```suggestion
	 * @param string        $css_template String containing the inline css to add.
```

**Pull request merged**